### PR TITLE
HPCC-26093 - Move metrics config from values.yml into example files

### PR DESCRIPTION
**Pull request by kenrowland**
Removed metrics config from values.yml. Created new examples
directory for metrics and added README and example yml files
for existing sinks.

Signed-off-by: Ken Rowland kenneth.rowland@lexisnexisrisk.com

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: helm/hpcc/values.yaml [hunk 1]
```diff
@@ -94,7 +86,7 @@ security:
     extern: "allow"
     datafile: "allow"
 
-## storage:
+## ge:
```
Comment (ghalliday) [inline helm/hpcc/values.yaml:89]: edited by accident.

**Review by ghalliday — COMMENTED** [review]
@kenrowland one minor mistake.  Also, are you expecting this to go in 8.2? if so please target 8.2.x..

File: helm/examples/metrics/loggingsink.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#
+# Defines a logging sink that writes metric reports to the log
```
Comment (rpastrana) [inline helm/examples/metrics/loggingsink.yml:2]: "the log" here seems a little ambiguous (perhaps by design?).
I'm not sure if this adds much, but perhaps something like:
"Defines a logging sink that reports metrics as standard component log entries" ?

**Comment by kenrowland** [comment]
@ghalliday 8.2 is probably the best choice because it will disable metrics by default. Enabling should be an active choice that includes where they are reported. I will retarget accordingly.

**Review by rpastrana — COMMENTED** [review]
@kenrowland just one minor comment.
Also, how will users know what metrics are available?
Have we thought about providing a mechanism which lists available metrics? (I suppose enabling metrics output, and setting up prometheus/ES just to see what's available isn't very user friendly)

**Pull request closed without merging**